rocketpy/utilities.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Franz Masatoshi Yuri"
+__copyright__ = "Copyright 20XX, Projeto Jupiter"
+__license__ = "MIT"
+
+import re
+import math
+import bisect
+import warnings
+import time
+from datetime import datetime, timedelta
+from inspect import signature, getsourcelines
+from collections import namedtuple
+
+import numpy as np
+from scipy import integrate
+from scipy import linalg
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+
+from rocketpy import Environment, Rocket, SolidMotor, Flight

```

Comment:
I believe these import statements are not needed, so I would suggest removing them.

```suggestion
```